commit 73a628a5cebfa6dae1fca4c5fde072376ce4d06f
Author: Georg Koppen <gk at torproject.org>
Date:   Fri Sep 12 09:37:39 2014 +0000

    Bug 12811: Add MOZ_D3DCOMPILER_DLL iff available.
    
    It seems MOZ_D3DCOMPILER_DLL is not avilable when compiling with
    mingw-w64. Trying to include it nevertheless breaks the packaging step
    as `@BINPATH@/@MOZ_D3DCOMPILER_DLL@` resolves to `bin/` which results
    in including all the things beginning with a `bin/` (again) which is
    treated as an error in case they got already included.
    
    This got fixed more or less by accident by bug 999260 which did not make
    it into ESR 31 (https://hg.mozilla.org/mozilla-central/rev/8b48386fc226).
---
 browser/installer/package-manifest.in |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/browser/installer/package-manifest.in b/browser/installer/package-manifest.in
index 521538f..c9117cf 100644
--- a/browser/installer/package-manifest.in
+++ b/browser/installer/package-manifest.in
@@ -597,8 +597,10 @@
 #ifdef MOZ_ANGLE_RENDERER
 @BINPATH@/libEGL.dll
 @BINPATH@/libGLESv2.dll
+#ifdef MOZ_D3DCOMPILER_DLL_PATH
 @BINPATH@/@MOZ_D3DCOMPILER_DLL@
 #endif
+#endif
 
 ; [Browser Chrome Files]
 @BINPATH@/browser/chrome.manifest
